lighthouse-core/test/fraggle-rock/api-test-pptr.js

```diff
@@ -95,7 +95,7 @@ describe('Fraggle Rock API', () => {
 
       const {auditResults, erroredAudits, failedAudits} = getAuditsBreakdown(lhr);
       // TODO(FR-COMPAT): This assertion can be removed when full compatibility is reached.
-      expect(auditResults.length).toMatchInlineSnapshot(`67`);
+      expect(auditResults.length).toMatchInlineSnapshot(`72`);
```

Comment:
There were 4 misspellings but this jumped from 67 to 72. Is that because more than 1 audit use the same artifact?

Reply:
yep!